### ISSUE TITLE
DB3 Migration Setup

### ISSUE THREAD

**Issue by ZachariahPepper** [Studio-Ops-Org/Studio-2022-S2-Repo#35]
Migrate the following databases and scripts from the DB3 server to the new SSMS server

- [x] -Northwind
- [x] -World
- [x] -Adventure Works
- [x] -BIT student database

**Comment by westcl4** [comment]
We copied over the database schemas however the data in the tables was not copied. We also had some bizarre connection issues; the dev server was able to get a response from pinging the prod server however the same did not work in the opposite direction. If we could establish a connection we could copy over the entire database using the Wizard built into the SSMS app. This is pending further investigation.

**Comment by eadsysadmin** [comment]
There have long been issues with the Copy Database Wizard in MS SQL, many of which relate to network/firewall/connectivity challenges. How long will you spend trying to resolve this connection issue? What other approaches could you use to move the databases? @s-winton @westcl4

**Comment by s-winton** [comment]
Yesterday we managed to generate schemas with insert statements and manually uploaded them to the dev SSMS, only 1 script was too big to load, which was the adventure works DB.

**Comment by westcl4** [comment]
Can we temporarily assign more RAM to the dev machine? It says it runs out of Memory for running this script. @eadsysadmin

**Comment by westcl4** [comment]
![image](https://user-images.githubusercontent.com/57237159/185515263-6f16709a-e06a-4508-ac6c-0ed6270cddba.png)

**Comment by eadsysadmin** [comment]
Yes. Add more memory. A dev server that won't run the core tasks is useless.

**Comment by westcl4** [comment]
![image](https://user-images.githubusercontent.com/57237159/185813183-c2b74da6-9eea-4aaa-aa0f-df604fd82acd.png)

Server has been resized temporarily.

**Comment by westcl4** [comment]
This error was thrown when we ran the AdventureWorks2019 query.

![image](https://user-images.githubusercontent.com/57237159/185813289-3fbdfce9-383f-4a44-9a3e-bf42593017fb.png)

**Comment by westcl4** [comment]
Navigating to this file reveals this error output:

![image](https://user-images.githubusercontent.com/57237159/185813411-66c6a004-4bc2-4b0e-94b1-914fe22cd0f4.png)
